travis/cfg/travis_run_pylint.cfg

```diff
@@ -6,7 +6,39 @@ cache-size=500
 
 [MESSAGES CONTROL]
 disable=all
-enable=E0101,E1124,E1306,E1601,I0013,W0101,W0102,W0104,W0105,W0109,W0403,W0404,W1111,W1401,W0402
+
+# Enable message and code:
+#   anomalous-backslash-in-string - W1401
+#   assignment-from-none - W1111
+#   dangerous-default-value - W0102
+#   deprecated-module - W0402
+#   duplicate-key - W0109
+#   file-ignored - I0013
+#   pointless-statement - W0104
+#   pointless-string-statement - W0105
+#   print-statement - E1601
+#   redundant-keyword-arg - E1124
+#   reimported - W0404
+#   relative-import - W0403
+#   return-in-init - E0101
+#   too-few-format-args - E1306
+#   unreachable - W0101
+
+enable=anomalous-backslash-in-string,
+    assignment-from-none,
+    dangerous-default-value,
+    deprecated-module,
+    duplicate-key,
+    file-ignored,
+    pointless-statement,
+    pointless-string-statement,
+    print-statement,
+    redundant-keyword-arg,
+    reimported,
+    relative-import,
+    return-in-init,
+    too-few-format-args,
+    unreachable,
```

Comment:
Is it allowed to add a comments at the end of each line? 
If so I suggest to add the message code at the end of each line and remove the comment block above.